aws-distro-opentelemetry-node-autoinstrumentation/test/aws-opentelemetry-configurator.test.ts

```diff
@@ -110,6 +116,107 @@ describe('AwsOpenTelemetryConfiguratorTest', () => {
     }
   });
 
+  describe('Propagator Extraction', () => {
+    const envVarsToRestore: NodeJS.ProcessEnv = {};
+    beforeEach(() => {
+      for (const [key, value] of Object.entries(process.env)) {
+        if (key.startsWith('OTEL_')) {
+          envVarsToRestore[key] = value;
+          delete process.env[key];
+        }
+      }
+    });
+
+    afterEach(() => {
+      // Clean-up
+      for (const [key, value] of Object.entries(envVarsToRestore)) {
+        process.env[key] = value;
+      }
+    });
+
+    it('Default Propagator extraction of Trace Context works if only X-Ray Trace Header is set', () => {
+      const carrier: Record<string, string> = {
+        'X-Amzn-Trace-Id':
+          'Root=1-5759e988-bd862e3fe1bf46a994270000;Parent=53945c3f42cd0000;Sampled=1;Lineage=a87bd80c:1|68fd508a:5|c512fbe3:2',
+      };
```

Comment:
is it necessary to add tests to cover the follow cases if not yet covered in tests 
1. only w3c header
2. both formats

Reply:
> Do we want to test that if Incoming HTTP Header has mismatching W3C trace Context and X-Ray Trace Context, X-Ray Trace Context always take precedence?

As we discussed, let's add the tests for this case if it is not existed in upstream. So we could detect if this expected behavior got changed.